acex.tex

```diff
@@ -2767,6 +2767,15 @@ \subsection{Error Messages}
 \item[\cword{message from acepho---photoelectric processing not complete}] ~\par
   This version doesn't handle fluorescence data as yet.
 
+\item[\cword{error in acepho***storage exceeded for the coherent form factors}] ~\par
+  The size for the scratch array \cword{scr} is too small. Increase the value for \cword{nwscr} (default value set to 50000).
+
+\item[\cword{error in acepho***storage exceeded for the incoherent scattering function}] ~\par
+  The size for the scratch array \cword{scr} is too small. Increase the value for \cword{nwscr} (default value set to 50000).
+
+\item[\cword{error in alax***storage exceeded for the atomic relaxation data}] ~\par
+  The size for the scratch array \cword{scr} is too small. Increase the value for \cword{nwscr} (default value set to 50000).
+
```

Comment:
Lines 2771, 2774, and 2777 are all exactly the same. Is that the intent?

Reply:
Yes, it is the same scratch array in all three cases, just in different spots in the program.